javacutil/src/main/java/org/checkerframework/javacutil/Resolver.java

```diff
@@ -262,6 +262,36 @@ public Element findClass(String name, TreePath path) {
         }
     }
 
+    /**
+     * Finds the class with {@code name} in a given type.
```

Comment:
Does "in a given type" mean this is looking for a nested class?  Saying that (and even adding "Nested" to the method name) would be clearer, I think.